LEGO1/mxtransitionmanager.cpp

```diff
@@ -370,7 +370,7 @@ void MxTransitionManager::SetupCopyRect(LPDDSURFACEDESC ddsc)
     // Allocate the copy buffer
     const char *src = (const char*)ddsc->lpSurface + m_copyRect.top * ddsc->lPitch + bytesPerPixel * m_copyRect.left;
 
-    m_copyBuffer = malloc(bytesPerPixel * width * height);
+    m_copyBuffer = new char[bytesPerPixel * width * height];
```

Comment:
We should probably change the type of `m_copyBuffer` from `void*` to `char*`

Reply:
MxU8* is even better